src/app/file-manager/file-explorer/file-explorer.component.ts

```diff
@@ -24,13 +24,16 @@ export class FileExplorerComponent implements OnInit {
   msgs: Message[];
 
   @Output('done') done: EventEmitter<any> = new EventEmitter();
+  @Output('count') count: EventEmitter<any> = new EventEmitter();
```

Comment:
I see that `'count`` is used by `TransferablesGridComponent` but it doesn't appear to be ever set with a value here. If it's not being used it should be removed.

Reply:
it is used to pass the selected count value from file-explorer.component.ts to transferable-grid.component.ts. It is set on home.component.html.

Reply:
I'm still learning angular so I'm probably missing something here, but I see `<app-file-explorer ... (count)="track($event)"` in `home.component.html`. My understanding was that this binds `count` as an output of `app-file-explorer`. But `count` (unlike `done`) doesn't appear to be set anywhere in `FileExplorerComponent` so its value will always be `0`. I tried setting a breakpoint on `HomeComponent.track()` and wasn't able to make it trigger.

Reply:
The emitter was never called, but now ii is removed and replaced with datasource.data.length number. Cleaner, shorter code, I guess.